src/applications/pensions/components/IntroductionPage.jsx

```diff
@@ -1,139 +1,297 @@
+        prefillEnabled={formConfig.prefillEnabled}
+        pageList={route.pageList}
+        downtime={route.formConfig.downtime}
+        startText="Start the pension application"
+        retentionPeriod="one year"
+        retentionPeriodStart="when you start"
+        continueMsg={<FormReactivationAlert />}
+      />
+      <h2 className="vads-u-font-size--h3 vad-u-margin-top--0">
+        Follow these steps to apply for a Veterans Pension
+      </h2>
+      <va-process-list uswds>
+        <va-process-list-item header="Prepare">
+          <h4 className="vads-u-margin-y--1">
+            To fill out this application, you’ll need this information:
+          </h4>
+          <ul>
+            <li>
+              Your Social Security number or VA file number{' '}
+              <span className="vads-u-color--secondary-dark">(*Required)</span>
+            </li>
+            <li>
+              Your military history{' '}
+              <span className="vads-u-color--secondary-dark">(*Required)</span>
+            </li>
+            <li>
+              Financial information about you and your dependents{' '}
+              <span className="vads-u-color--secondary-dark">(*Required)</span>
+            </li>
+            <li>Your marital status and prior marital history</li>
+            <li>Information about your spouse’s prior marriage</li>
+            <li>Information about your dependent children</li>
+            <li>Your employment history</li>
+          </ul>
+          <h4>
+            If you have special circumstances for your medical care, you may
+            also need these additional forms:
+          </h4>
+          <ul>
+            <li>
+              <strong>Statement of Medical Care:</strong> Care Worksheets at the
+              end of this pension benefits form that must be completed by an
+              administrator or licensed medical professional
+            </li>
+            <li>
+              Claim for Special Monthly Pension (
+              <a
+                href="https://www.vba.va.gov/pubs/forms/VBA-21-2680-ARE.pdf"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                VA Form 21-2680
+              </a>
+              )
+            </li>
+            <li>
+              Claim for Medicare Nursing Home and/por $90.00 Rate Reduction
+              Request (
+              <a
+                href="https://www.vba.va.gov/pubs/forms/VBA-21-0779-ARE.pdf"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                VA Form 21-0779
+              </a>
+              )
+            </li>
+            <li>
+              Claim for Fiduciary Assistance (
+              <a
+                href="https://www.vba.va.gov/pubs/forms/VBA-21-2680-ARE.pdf"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                VA Form 21-2680
+              </a>
```

Comment:
This is the same form as the one above for `claim for special monthly pension`, is this intentional?

Reply:
I'm assuming it is since this is part of the Introduction page that has been through staging review and is in production. I will seek clarification.